fetch-utils.js

```diff
@@ -4,27 +4,35 @@ const SUPABASE_KEY =
 const client = supabase.createClient(SUPABASE_URL, SUPABASE_KEY);
 
 export async function getCountries(name, continent) {
-    // > Part D: Add a second argument to `select(` to 
+    // > Part D: Add a second argument to `select(` to
     // return an exact db count of matching records
 
     // > Part A: Implement the client query from countries:
     //   1. select all columns
     //   2. order by country name
     //   3. limit to 100 countries
-    let query = // ?
+    let query = client.from('countries').select('*', { count: 'exact' }).order('name').limit(100);
```

Comment:
this syntax took a while to click for me. from and select are the big methods to remember. also just keep in mind these are supabase methods that use JavaScript not JavaScript methods (if that makes sense)